src/Symfony/Component/Notifier/Bridge/FakeSms/composer.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/fake-sms-notifier",
+    "type": "symfony-bridge",
+    "description": "Fake SMS (as email during development) Notifier Bridge.",
+    "keywords": ["sms", "development", "email", "notifier", "symfony"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "James Hemery",
+            "homepage": "https://github.com/JamesHemery"
+        },
+        {
+            "name": "Yield Studio",
+            "homepage": "https://github.com/YieldStudio"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.1",
```

Version 4.3 and 5.1 is not supported anymore. 

```suggestion
        "symfony/http-client": "^4.4|^5.2",
```